tests/service/test_dev.py

```diff
@@ -322,9 +322,9 @@ async def test_dep_check(self):
 
 class TestPinDeps(AsyncTestCase):
     @cached_download(
-        "https://github.com/intel/dffml/files/5587450/l.txt",
+        "https://github.com/intel/dffml/files/5835924/l.txt",
         pathlib.Path(__file__).parent / "logs.txt",
-        "fd16470e45f076550507c015cc102387a2627547cbe8b64753f7fa0f6f273a26c59e619bd7a06b49613c6e574839a08a",
+        "e8b33fd0cc297df7c831135d5b0d8e2d7c5575059e1e69feae8cee9cd7dff7212c6454e8e0ba93b33fe78f3de0ba35fc",
```

Comment:
@nitesh585 I just removed this test. Could you update the PR with master and I'll merge? Thanks!

Reply:
done. Thanks :)